src/ameba/rule/style/method_names.cr

```diff
@@ -46,17 +48,10 @@ module Ameba::Rule::Style
 
     def test(source, node : Crystal::Def)
       return if (expected = node.name.underscore) == node.name
+      return unless (location = name_location(node))
+      return unless (end_location = name_end_location(node))
```

Comment:
```suggestion
      return unless location = name_location(node)
      return unless end_location = name_end_location(node)
```

ditto elsewhere

Reply:
Are there any plans for an Ameba rule for parenthesized assignments similar to RuboCop’s [`Lint/AssignmentInCondition`](https://docs.rubocop.org/rubocop/cops_lint.html#lintassignmentincondition)? The setting `AllowSafeAssignment` defaults to `true`, but would the equivalent Ameba setting default to `false` instead?

Reply:
I don’t think it is planned yet. The assignment in condition is much often that in ruby. We can discuss this in a separate issue